pco/src/float_mult_utils.rs

```diff
@@ -651,25 +628,37 @@ mod test {
       junk.push((i as f32).sin());
     }
 
+    let sevenths_sample = &sevenths[..50];
+
     assert_eq!(
       choose_config(&sevenths),
       Some(FloatMultConfig {
         base: 1.0 / 7.0,
         inv_base: 7.0,
       })
     );
-    assert_eq!(choose_config(&ones), None);
+    assert_eq!(
+      choose_config(&ones),
+      Some(FloatMultConfig {
+        base: 1.0,
+        inv_base: 1.0,
```

Comment:
Are there any unit tests that would fail if `inv_base` was not correctly plumbed through the library code (i.e. got clobbered by `base.inv()` somewhere)?

Reply:
Hmm, probably not, but I have unit tests for `snap_to_int_reciprocal`. To test this we'd want some multiples of e.g. 1/7, some of which are extremely large such that 1.0/7.0 ~= 1ULP

Reply:
Do you think it's worth adding?